### DISCUSSION TITLE
Support for nulls_first / nulls_last in CursorPagination

### DISCUSSION THREAD

**Discussion by jonatanvm** [encode/django-rest-framework#9456] [Ideas & Suggestions]
Django supports [asc(nulls_first=None, nulls_last=None)](https://docs.djangoproject.com/en/5.0/ref/models/expressions/#django.db.models.Expression.asc) and [desc(nulls_first=None, nulls_last=None)](https://docs.djangoproject.com/en/5.0/ref/models/expressions/#django.db.models.Expression.desc), but DRF does not. 
Even if you write your own OrderingFilter which orders the results with nulls last, CursorPagination will override that in:
```
if reverse:
    queryset = queryset.order_by(*_reverse_ordering(self.ordering))
else:
    queryset = queryset.order_by(*self.ordering)
```

My suggestion is to add a configurable class variable that let's us configure whether nulls will be sorted first or last. Alternatively it would be helpful if ordering was extracted to a function that could be overridden.

**Comment by yangchoi** [comment]
I was also looking to discuss the same feature. If this feature is added, it will be a great help.

**Comment by sevdog** [comment]
This could be achieved by using [`Queryset.reverse()`](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#django.db.models.query.QuerySet.reverse) instead thant providing the method `_reverse_ordering`.

However it would be very nice if the `OrderingField` would have some kind of built-in support for Django ordering options for NULLs.
